pkgs/tools/networking/rpki-client/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib, stdenv, fetchurl, autoreconfHook, expat, libressl, rsync }:
+
+stdenv.mkDerivation rec {
+  pname = "rpki-client";
+  version = "8.0";
+
+  src = fetchurl {
+    url = "https://cdn.openbsd.org/pub/OpenBSD/${pname}/${pname}-${version}.tar.gz";
+    hash = "sha256-W3EMzuLn6UlYflTa+COBFnEXSlDGcXRuWidq+qDOVb4=";
+  };
+
+  propagatedBuildInputs = [
+    expat
+    libressl
+    rsync
+  ];
+
+  nativeBuildInputs = [
+    autoreconfHook
+  ];
+
+  configureFlags = [
+    "--with-output-dir=/var/lib/rpki-client"
+    "--localstatedir=/var"
+    "--with-user=rpki-client"
+  ];
+
+  meta = with lib; {
+    description = "OpenBSD's rpki-client";
```

Comment:
And what is an rpki client?

Reply:
We can spell the acronym out, but that will not help anyone. This is fine as is.